docs/UserGuide.md

```diff
@@ -3,172 +3,336 @@ layout: page
+create a seamless user experience for you. Telemarketers and telemarketer supervisors, such as yourselves, can look
+through this User Guide and try out some of the available features!
+
+[⏫ Back to content page](#table-of-contents)
 
-**_Command-line Interface (CLI)_**: A user interface that allows users to interact with a system through text commands.
+### 2.2. Structure of this User Guide
 
-**_Graphical User Interface (GUI)_**: A user interface that allows users to interact with a system through graphical icons.
+There are a lot of features included in Teletubbies. As a result, we have decided to structure our User Guide in a
+manner that allows you to easily find what you are looking for, without getting overwhelmed.
 
-**_JavaScript Object Notation (JSON)_**: A text-based representation of structured data that is used to save contacts as files in *Teletubbies*.
+In [Section 1: Introduction](#1-introduction), you will find a general overview of what Teletubbies is about.
 
-**_Parameter_**: Input to a command's field.
+In [Section 2: About this User Guide](#2-about-this-user-guide), you will find tips and advice on how to read and
+navigate through this document.
 
-### Glossary of icons and symbols
-:information_source: : Additional information <br/>
-:bulb: : Tip <br/>
-:exclamation: : Important <br/>
-:x: : Error / Avoid <br/>
+In [Section 3: Quick Start](#3-quick-start), you will find a step-by-step guide to help you get started with using
+Teletubbies.
+
+In [Section 4: Features](#4-features), you will find documentation on all of Teletubbies' commands and features.
+
+In [Section 5: FAQ](#5-faq), you will find the answers to some of the most frequently asked questions regarding
+Teletubbies.
+
+In [Section 6: Command Summary](#6-command-summary), you will find a summarised table of commands available in
+Teletubbies.
+
+In [Section 7: Glossary](#7-glossary), you will find the meanings and definitions of some of the more technical terms used in this
+document.
+
+Each of these sections and their sub-sections are accessible through the Table of Contents, which can be found at the
+beginning of this document.
 
 <div markdown="block" class="alert alert-info">
 
-**:information_source: Information block**<br>
+**:information_source: Note:**<br>
 
-* We use these blocks to enhance the readability of this guide!
+We have divided and categorised the [Features section](#4-features) into 4 categories in order to give you
+a bird’s eye view of some of the features available! You can peek into [System commands](#41-system-commands) to take a
+look at some of the system-level features available such as [help](#viewing-help-help) and
+[import](#importing-contacts-import). [Contact commands](#42-contact-commands) includes all the commands which allow you to
+modify the contacts in your contact list, while [List commands](#43-list-commands) includes commands to modify the list of
```

Comment:
could say instead: 'to view a more specific list of contacts' owtte. currently sounds a bit confusing immediately after the previous line